modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -314,6 +314,26 @@ def width(self):
             self._width_cache = self.apply(self._width_extraction_fn()).get()
         return self._width_cache
 
+    def split(self, split_func, num_splits, *args):
+        """
+        Split the object wrapped by the partition into multiple partitions.
+
+        Parameters
+        ----------
+        split_func : Callable[pandas.DataFrame, List[Any]] -> List[pandas.DataFrame]
+            The function that will split this partition into multiple partitions.
```

Comment:
I wonder if it's possible to sprinkle a little bit of details on what this `List[Any]` argument is - how it is describing the splits maybe?

Reply:
The `List[Any]` is going to be the same dtype as the column - since it will be pivot points - maybe I could add that to the description? I don't think changing the type annotation makes sense though.

Reply:
No longer an issue, since we just pass the NamedTuple!